packages/my-account/src/pages/SubscriptionsPage/index.tsx

```diff
@@ -0,0 +1,182 @@
+import { OrderList } from "@commercelayer/react-components/orders/OrderList"
+import { OrderListEmpty } from "@commercelayer/react-components/orders/OrderListEmpty"
+import { OrderListPaginationButtons } from "@commercelayer/react-components/orders/OrderListPaginationButtons"
+import { OrderListPaginationInfo } from "@commercelayer/react-components/orders/OrderListPaginationInfo"
+import { OrderListRow } from "@commercelayer/react-components/orders/OrderListRow"
+import { useContext } from "react"
+import { Trans, useTranslation } from "react-i18next"
+
+import {
+  OrderListWrapper,
+  OrderNumber,
+  OrderDate,
+  SubscriptionFrequency,
+} from "./styled"
+
+import Empty from "#components/composite/Empty"
+import SubscriptionStatusChip from "#components/composite/Subscription/SubscriptionStatusChip"
+import { SkeletonMainSubscriptionsTable } from "#components/composite/Skeleton/Main/SubscriptionsTable"
+import Title from "#components/ui/Title"
+import { AppContext } from "#providers/AppProvider"
+import { formatDate, shortDate } from "#utils/dateTimeFormats"
+import SubscriptionNextRunProgress from "#components/composite/Subscription/SubscriptionNextRunProgress"
+
+function SubscriptionsPage(): JSX.Element {
+  const { t } = useTranslation()
+  const ctx = useContext(AppContext)
+  const accessToken = ctx?.accessToken
+
+  const colClassName =
+    "text-left text-xs font-thin border-b border-gray-200 md:border-none text-gray-300 md:font-semibold md:uppercase md:relative"
+  const titleClassName = "flex gap-2"
+  const columns = [
+    {
+      header: "Subscription",
```

Comment:
headers need to be translated?

Reply:
Thanks for this one! I'll take care of translating also the ones of the order list.. Never noticed it!